neuro_extras/data.py

```diff
@@ -0,0 +1,382 @@
+import asyncio
+import logging
+import sys
+import tempfile
+from distutils import dir_util
+from enum import Enum
+from pathlib import Path
+from typing import List, Sequence
+
+import click
+from neuromation import api as neuro_api
+from neuromation.api.url_utils import uri_from_cli
+from neuromation.cli.asyncio_utils import run as run_async
+from neuromation.cli.const import EX_PLATFORMERROR
+from yarl import URL
+
+from .cli import main
+from .data_copier import NEURO_EXTRAS_IMAGE, DataCopier
+
+
+logger = logging.getLogger(__name__)
+
+SUPPORTED_ARCHIVE_TYPES = (
+    ".tar.gz",
+    ".tgz",
+    ".tar.bz2",
+    ".tbz2",
+    ".tbz",
+    ".tar",
+    ".gz",
+    ".zip",
+)
+TEMP_UNPACK_DIR = Path.home() / ".neuro-tmp"
+
+
+class UrlType(Enum):
+    UNSUPPORTED = 0
+    LOCAL = 1
+    CLOUD = 2
+    STORAGE = 3
+    DISK = 4
+
+    @staticmethod
+    def get_type(url: URL) -> "UrlType":
+        if url.scheme == "storage":
+            return UrlType.STORAGE
+        if url.scheme == "":
+            return UrlType.LOCAL
+        if url.scheme in ("s3", "gs"):
+            return UrlType.CLOUD
+        if url.scheme == "disk":
+            return UrlType.DISK
+        return UrlType.UNSUPPORTED
+
+
+@main.group()
+def data() -> None:
+    pass
+
+
+@data.command("transfer")
+@click.argument("source")
+@click.argument("destination")
+def data_transfer(source: str, destination: str) -> None:
+    run_async(_transfer_data(source, destination))
+
+
+@data.command("cp")
```

Comment:
cp? or transfer?

Reply:
this one is `cp`, the other one is `transfer`, we have both now

Reply:
OMG I thought we left only `neuro-extras data/image transfer`